src/database/utilities/createRecord.js

```diff
@@ -191,6 +191,7 @@ function createStocktakeBatch(database, stocktakeItem, itemBatch) {
     stocktake: stocktakeItem.stocktake,
     itemBatch: itemBatch,
     snapshotNumberOfPacks: numberOfPacks,
+    countedNumberOfPacks: numberOfPacks, // countedNumberOfPacks === snapshotNumberOfPacks
```

Comment:
I'd rather an explanation somewhere why the 2 fields are equal.

Reply:
I think they should be equal by default, otherwise need to check for null everywhere, and if null assume countedNumberOfPacks === snapshotNumberOfPacks.. And also need to check difference between null and 0.  I am not sure why we originally didn't assume countedNumberOfPacks === snapshotNumberOfPacks when stocktakeBatch is created.